08-transport.md

```diff
@@ -329,10 +323,9 @@ for the `poly1305` tag.
   * `ck, temp_k = HKDF(ck, ss)`
      * This phase generates a new temporary encryption key (`temp_k`) which is
        used to generate the authenticating MAC.
-     * The nonce `n` should be reset to zero: `n = 0`.
 
 
-  * `c = encryptWithAD(temp_k, n, h, zero)`
```

Comment:
So within this diff  `encryptWithAD/decryptWithAD` should always be called with a nonce of zero during the handshake.